packages/grafana-ui/src/components/Card/Card.tsx

```diff
@@ -100,66 +63,41 @@ export const Card: CardInterface = ({
   const disableHover = disabled || (!onClick && !href);
   const disableEvents = disabled && !actions;
 
-  const containerStyles = getContainerStyles(theme, disableEvents, disableHover);
   const onCardClick = useCallback(() => (disableHover ? () => {} : onClick?.()), [disableHover, onClick]);
 
   return (
     <CardContainer
       tooltip={tooltip}
       tabIndex={disableHover ? undefined : 0}
-      className={cx(containerStyles, className)}
       onClick={onCardClick}
+      disableEvents={disableEvents}
+      disableHover={disableHover}
+      href={href}
       {...htmlProps}
     >
-      <CardInner href={href}>
-        {figure}
-        <div className={styles.inner}>
-          <div className={styles.info}>
-            <div>
-              <div className={styles.heading} role="heading">
-                {heading}
-              </div>
-              {meta}
-              {description && <p className={styles.description}>{description}</p>}
+      {figure}
+      <div className={styles.inner}>
+        <div className={styles.info}>
+          <div>
+            <div className={styles.heading} role="heading">
+              {heading}
             </div>
-            {tags}
+            {meta}
+            {description && <p className={styles.description}>{description}</p>}
           </div>
-          {hasActions && (
-            <div className={styles.actionRow}>
-              {actions}
-              {secondaryActions}
-            </div>
-          )}
+          {tags}
         </div>
-      </CardInner>
+        {hasActions && (
+          <div className={styles.actionRow}>
+            {actions}
+            {secondaryActions}
+          </div>
+        )}
+      </div>
     </CardContainer>
   );
 };
 
-const getContainerStyles = stylesFactory((theme: GrafanaThemeV2, disabled = false, disableHover = false) => {
-  return css({
-    display: 'flex',
-    width: '100%',
-    background: theme.colors.background.secondary,
-    borderRadius: theme.shape.borderRadius(),
-    position: 'relative',
-    pointerEvents: disabled ? 'none' : 'auto',
-    marginBottom: theme.spacing(1),
-    transition: theme.transitions.create(['background-color', 'box-shadow', 'border-color', 'color'], {
-      duration: theme.transitions.duration.short,
-    }),
-
-    ...(!disableHover && {
-      '&:hover': {
-        background: theme.colors.emphasize(theme.colors.background.secondary, 0.03),
-        cursor: 'pointer',
-        zIndex: 1,
-      },
-      '&:focus': styleMixins.getFocusStyles(theme),
-    }),
-  });
-});
```

Comment:
An alternative idea would be to make this into a mixin that returns a CSSObject that we can reuse in other Card like elements, a CSSObject feels easier and cleaner to re-use & extend compared to a css class, What elements is it you want to share for your new card? the background, hover & focus state? the inner link?

Reply:
It's for report list view, I think `CadContainer` works pretty good, it also abstracts the logic that inner content can be a link or a div. 
![Screenshot 2021-04-28 at 9 43 02](https://user-images.githubusercontent.com/8878045/116358275-2214f100-a806-11eb-9345-048a976957e2.png)

Reply:
looks more like table :)

Reply:
yeah, it used to be a table, but I thought it's easier to style it if it's just a Card item.